Jenkinsfile_CNP

```diff
@@ -64,8 +66,8 @@ node {
                         onMaster {
 //                                withSubscription('nonprod') {
 //
-//                                        KeyVault keyVault = new KeyVault(this, 'nonprod', "ccd-${environment.nonProdName}")
```

Comment:
not sure why the block of code for AAT deployment is commented out but this is not correct. Can you please uncomment so that we deploy on AAT and Prod ?